### PR TITLE
`update-drafts` script: don't make date-only modifications

### PR DESCRIPTION
**Pull request by ddbeck**
~~Blocked by https://github.com/web-platform-dx/web-features/pull/1679, then a rebase is needed.~~

This skips writing changes to draft features when the only change is the `draft_date` value. This prevents needlessly large diffs when updating these files and makes it possible to find out which areas we've touched least recently.
    
Fixes https://github.com/web-platform-dx/web-features/issues/1716.

### CODE REVIEW

File: scripts/update-drafts.ts [hunk 1]
```diff
@@ -178,7 +196,43 @@ async function main() {
...
+
+    // If there's a file for this spec already, write updates only if something
+    // other than the `draft_date` changed. Because changes can be comments, we
+    // have to check a diff rather than parsing and comparing values.
+    const changes = diff.diffLines(originalFile, proposedFile);
+    const noChanges =
+      changes.length === 1 && !changes[0].added && !changes[0].removed;
+    const onlyDateChanges =
+      !noChanges &&
+      changes.length === 3 &&
```
Comment (autonome) [inline scripts/update-drafts.ts:224]: document the magic 1 and 3 numbers above

**Review by autonome — APPROVED** [review]
looks good to me, just a note on some inline docs that'd be helpful to future readers

File: scripts/update-drafts.ts (hunk 1 above)
Comment (ddbeck) [reply scripts/update-drafts.ts:230]: Good idea! I won't even remember that in a week. 😅 Fixed with 13e94dbc.

**Review by Elchi3 — APPROVED** [review]
Works as expected for me 👍

**Pull request merged**